### PR TITLE
Update k8s to 0.26.2 and OpenShift to 4.13

### PR DESCRIPTION
**Pull request by Red-GV**
This PR updates the k8s dependencies to v0.26.2 and OpenShift dependencies to 4.13.

/cc @xperimental 
/assign @periklis 

- JIRA: https://issues.redhat.com/browse/LOG-3686

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,43 +5,33 @@ go 1.19
...
 	github.com/ViaQ/logerr/v2 v2.0.0
 	github.com/go-logr/logr v1.2.3
-	github.com/google/go-cmp v0.5.8
+	github.com/google/go-cmp v0.5.9
 	github.com/inhies/go-bytesize v0.0.0-20151001220322-5990f52c6ad6
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.20.1
-	github.com/openshift/api v0.0.0-20220712151050-2647eb31dee7 // Corresponds to release-4.11
+	github.com/onsi/gomega v1.24.1
+	github.com/openshift/api v0.0.0-20230228142948-d170fcdc0fa6 // Corresponds to release-4.11
```
Comment (periklis) [inline go.mod:12]:
```suggestion
	github.com/openshift/api v0.0.0-20230228142948-d170fcdc0fa6 // Corresponds to release-4.13
```

**Review by periklis — CHANGES_REQUESTED** [review]
LGTM to me, a couple of things though:
1. What's about the Dockerfile's using go1.20? Follow-up PR?
2. Doesn't the Makefile need for `operator-sdk run bundle` the new `--security-context restricted` option to be set?

**Comment by Red-GV** [comment]
Yes @periklis, I think we will need to wait for this PR (https://github.com/openshift/release/pull/37343) and all the images to be published.

**Review by periklis — APPROVED** [review]

**Pull request merged**